tests/tests_pytorch/trainer/connectors/test_accelerator_connector.py

```diff
@@ -576,6 +576,7 @@ def mock_hpu_available(monkeypatch, value=True):
     monkeypatch.setattr(lightning_habana.plugins.precision, "_HPU_AVAILABLE", value)
 
 
+@pytest.mark.skipif(not package_available("lightning"), reason="Supported only with mono-package")
```

Comment:
why this?

Reply:
Jirka added it I think when it was inside the pytroch folder

Reply:
so we can remove it?

Reply:
yes